makefile

```diff
@@ -5,13 +5,24 @@ WINDOWS_BUILD_PATH = $(BUILD_PATH)/windows/idl.exe
 MAC_BUILD_PATH = $(BUILD_PATH)/darwin/idl
 
 export VERSION=$(shell gogitver)
+export COMMIT_HASH=$(shell git rev-parse HEAD)
+export BUILD_DATE=$(shell date +%Y-%m-%dT%T%z)
+export BUILD_FLAGS=\
+	-X github.com/syncromatics/idl-repository/cmd/idl/cmd.version=$(VERSION) \
```

Comment:
I gotta remember this one. Awesome way to get the version in there.

Reply:
go-version turned me on to build variables: https://github.com/christopherhein/go-version

It's pretty cool, but it took me a while to find the docs that led to using the full package/import path to make it work.